cmd/profile.go

```diff
@@ -66,6 +66,15 @@ goc profile --force
 		if output == "" {
 			fmt.Fprint(os.Stdout, string(res))
 		} else {
+
+			if dir != "" {
```

Comment:
Good requirements, but no need to add new flag. It's better to support like these:
`goc profile -o /path/to/file`
`goc profile -o relative-path/to/file`

Reply:
I think

-o: Write the log information to the specified file
-d: Save the file to the specified path or create it if it does not exist

They have different functions

Reply:
i prefer the behavior following command `go build [-o output]`, refer: https://golang.org/cmd/go/#hdr-Compile_packages_and_dependencies

Reply:
i prefer the behavior following command `go build [-o output]`, refer: https://golang.org/cmd/go/#hdr-Compile_packages_and_dependencies

Reply:
Thank you. I understand